externs/olx.js

```diff
@@ -623,6 +623,12 @@ olx.OverlayOptions.prototype.autoPanAnimation;
  */
 olx.OverlayOptions.prototype.autoPanMargin;
 
+/**
+ * CSS class name. Default is `ol-overlay-container ol-selectable`.
+ * @type {string|undefined}
+ * @api
+ */
+olx.OverlayOptions.prototype.className;
```

Comment:
please also add this in the `olx.OverlayOptions` `@typedef` above

Reply:
Fixed!